dcli/generatorCli.py

```diff
@@ -128,3 +130,20 @@ def python(self):
         common_end_message(answers)
         pass
 
+    def kotlin(self):
+        """ Generates a basic Kotlin challenge """
+        answers = ask_usual()
+        answers.update(askKotlinQuestions())
+
+        language = Kotlin()
+        self.add_type_if_necessary(self.parse_target_method_args(answers['targetMethodArgs']), language)
+        self.add_type_if_necessary(self.parse_target_method_args(answers['targetMethodReturn']), language)
```

Comment:
There is bug, when I generate a challenge with:  
Input: amount: Int
Output: String

I have a file called `amount:.kt` within the _success_ and _template_ folder.

Reply:
Done